### ISSUE TITLE
Aeneas for music?

### ISSUE THREAD

**Issue by chikiuso** [readbeyond/aeneas#209]
Hi, is there any way we could align lyrics to music audio with Aeneas? I just have a try and the result is not good.

**Comment by readbeyond** [comment]
Hi,

please note that I stated explicitly that aeneas is not fit for lyrics 
alignment ("song captioning"): 
https://github.com/readbeyond/aeneas/#limitations-and-missing-features

The algorithmic approach of aeneas is just not suitable for that in 
general. Depending on the kind of background music, you might be able to 
isolate it from speech, and then use aeneas to process the isolated 
speech. For example, I had a try for some opera, and it was okay. Rap 
might be too hard. Etc.

For what matters and as far as I know, aligning speech over music is 
still an open topic in the speech research --- so even more complex 
algorithmic approaches are far from producing good results --- but I can 
imagine that the current wave of neural network -fueled research will 
lead to developments for this problem soon.

So, in short: aeneas is not fit for lyrics alignment over generic music.
